### PR TITLE
updated icub nancy urdf for balancing and standup

### PR DESCRIPTION
**Pull request by gabrielenava**
I updated the model of iCub Nancy that now coincides with the one inside [icub-models](https://github.com/robotology-playground/icub-models). Furthermore, I added a model containing also the frames for standup demo (and I verified their pose using rviz)

### CODE REVIEW

**Comment by traversaro** [comment]
Can you add a short `README.md` to the `iCubNancy01` directory describing the different models? Just simple links to the appropriate GitHub issues will be enough, thanks!

**Comment by traversaro** [comment]
ping @gabrielenava

**Comment by gabrielenava** [comment]
done at https://github.com/robotology/yarp-wholebodyinterface/pull/80/commits/e91081ac49a3c51912f310d00fa6a9f2749b1195

File: app/robots/iCubNancy01/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# iCub Nancy models
+
+## List of models
+
+- model.urdf --> it is the one loaded in yarpWholeBodyInterface.ini
+
+- model__BALANCING_IIT.urdf --> it is the model used for YOGA++ demo (the same of https://github.com/robotology-playground/icub-models/tree/master/iCub/robots/iCubNancy01)
```
Comment (traversaro) [inline app/robots/iCubNancy01/README.md]: model__BALANCING_IIT --> model_BALANCING_IIT

File: app/robots/iCubNancy01/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+- model__STANDUP.urdf --> it is used for the standup demo (added upper leg contact frames, see https://github.com/robotology-playground/icub-model-generator/issues/42)
```
Comment (traversaro) [inline app/robots/iCubNancy01/README.md]: model__STANDUP -> model_STANDUP

**Comment by traversaro** [comment]
I am confused by the models committed in this PR. 

`model.urdf` and `model_STANDUP.urdf` are identifical. Furthermore, `model_BALANCING_IIT.urdf` is almost identical to the other two files, with some trivial unrelated modification in the names of the meshes, and with the `(l|r)_upper_leg_contact` frames removed. If the models are identical modulo some additional frames, why not keep a single `model.urdf`? 

Just to clarify, f2f and in https://github.com/robotology/yarp-wholebodyinterface/pull/78#issuecomment-302666016 I proposed to use a separated model for the balancing/standup for iCubNancy based on iCubDarmstadt01 because, while iCubNancy has v1 forearms, I trust better the accuracy of the simmechanics-generated model for the legs and the torso kinematics and dynamics parameters. Furthermore, as far as I remember we extracted the location of the `(l|r)_upper_leg_contact` frame for simmechanics-generated models, so I do not really understand where do you obtained the location of that frame w.r.t. to the (l|r)_upper_leg link frame for dh-generated models such as the iCubNancy01 one.

File: app/robots/iCubNancy01/README.md [hunk 3]
```diff
@@ -1,9 +1,10 @@
 # iCub Nancy models
 
-## List of models
+To better deal with local modifications of iCub Nancy model, the file model.urdf has been duplicated. In particular:
 
-- model.urdf --> it is the one loaded in yarpWholeBodyInterface.ini
+- model.urdf --> it is the file loaded in yarpWholeBodyInterface.ini and is the urdf that can be found in https://github.com/robotology-playground/icub-models/tree/master/iCub/robots/iCubNancy01, plus two additional frames for the upper legs (see also https://github.com/robotology-playground/icub-model-generator/issues/42);
 
-- model_BALANCING_IIT.urdf --> it is the model used for YOGA++ demo (the same of https://github.com/robotology-playground/icub-models/tree/master/iCub/robots/iCubNancy01)
+- model_STANDUP.urdf --> it is the model used for the standup demo. For the time being, it is identical two model.urdf;
```
Comment (traversaro) [inline app/robots/iCubNancy01/README.md]: identical two model.urdf; --> identical to

**Pull request merged**